packages/backend/src/core/discovery/utils/diffToMessage.ts

```diff
@@ -1,26 +1,27 @@
 import { DiscoveryDiff } from './diffDiscovery'
 
-export function diffToMessage(name: string, diff: DiscoveryDiff[]): string[] {
-  const beginning = `Detected changes for ${name}\n\n\`\`\`diff\n`
-  const end = '```'
+export function diffToMessage(name: string, diffs: DiscoveryDiff[]): string[] {
+  const prefix = `Detected changes for ${name}\n\n\`\`\`diff\n`
+  const postfix = '```'
```

Comment:
This ``` in prefix and postfix (by the way it should probably be suffix) could be extracted to a function called wrapCodeBlock(message: string). This would improve readability.

Reply:
but what about overheadLenght?

Reply:
hmmm, maybe you can extract it to prefix and content where content would be wrapCodeBlock(diff) then there is no need for suffix